### PR TITLE
Import vol doc

### PR DESCRIPTION
**Pull request by ansarars**

### CODE REVIEW

**Comment by datamattsson** [comment]
Just a FYI, @c-snell is on PTO this week. Don't expect a review until next week.

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 1]
```diff
@@ -141,6 +142,17 @@ These parameters are applicable only for Pod inline volumes and to be specified
...
+### Import volume
+In Import volume process, legacy or docker/k8s created volume will be renamed with prefix “pvc-“ and exposed to CSI driver. All previous Access Control Records and Initiator Groups will be stripped from the volume when volume is exposed to HPE CSI Driver.
+
+| Parameter          | Option  | Description |
+| ------------------ | ------- | ----------- |
+| accessProtocol     | fc or iscsi  | The access protocol to use when accessing the persistent volume. |
+| importVol          | Text    | The name of the 3PAR or Primera volume to import. |
+
+!!! important
+    **No other parameters** are required in Storage Class for import volume **except** parameters mentioned in above table.
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:154]: `StorageClass`

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 2]
```diff
@@ -141,6 +142,17 @@ These parameters are applicable only for Pod inline volumes and to be specified
 !!! important
     All parameters are **required** for inline ephemeral volumes.
 
+### Import volume
+In Import volume process, legacy or docker/k8s created volume will be renamed with prefix “pvc-“ and exposed to CSI driver. All previous Access Control Records and Initiator Groups will be stripped from the volume when volume is exposed to HPE CSI Driver.
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:146]: Grammar: 
- During the import volume process, any legacy (non-container volumes) or existing docker/k8s volume defined in the **ImportVol** parameter, within a `StorageClass`, will be renamed to match the `PersistentVolumeClaim` that leverages the `StorageClass`. The new volumes will be exposed through the HPE CSI Driver and made available to the Kubernetes cluster. **Note:** All previous Access Control Records and Initiator Groups will be removed from the volume when it is imported.

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 3]
```diff
@@ -151,6 +163,14 @@ How to use `VolumeSnapshotClass` and `VolumeSnapshot` objects is elaborated on i
 | ----------- | ------  | ----------- |
 | read_only   | Boolean | Indicates if the snapshot is writable on the 3PAR or Primera array. |
 
+### Import Snapshot
+In Import snapshot process, legacy or docker/k8s created snapshot will be renamed with prefix “snapshot-“ and exposed to CSI driver. All previous Access Control Records and Initiator Groups will be stripped from the snapshot when snapshot is exposed to HPE CSI Driver.
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:167]: During the import snapshot process, any legacy (non-container snapshot) or an existing docker/k8s snapshot defined in the ImportVol parameter, within a StorageClass, will be renamed with the prefix "snapshot-". The new snapshot will be exposed through the HPE CSI Driver and made available to the Kubernetes cluster. Note: All previous Access Control Records and Initiator Groups will be removed from the snapshot when it is imported.

File: docs/container_storage_provider/hpe_3par_primera/index.md (hunk 1 above)
Comment (ansarars) [reply docs/container_storage_provider/hpe_3par_primera/index.md]: Addressed this comment

File: docs/container_storage_provider/hpe_3par_primera/index.md (hunk 2 above)
Comment (ansarars) [reply docs/container_storage_provider/hpe_3par_primera/index.md]: Addressed this comment

File: docs/container_storage_provider/hpe_3par_primera/index.md (hunk 3 above)
Comment (ansarars) [reply docs/container_storage_provider/hpe_3par_primera/index.md]: Addressed this comment

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 4]
```diff
@@ -141,6 +142,17 @@ These parameters are applicable only for Pod inline volumes and to be specified
 !!! important
     All parameters are **required** for inline ephemeral volumes.
 
+### Import volume
+During the import volume process, any legacy (non-container volumes) or existing docker/k8s volume defined in the **ImportVol** parameter, within a StorageClass, will be renamed to match the PersistentVolumeClaim that leverages the StorageClass. The new volumes will be exposed through the HPE CSI Driver and made available to the Kubernetes cluster. **Note:** All previous Access Control Records and Initiator Groups will be removed from the volume when it is imported.
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:146]: small tweak: highlight Kubernetes objects with ticks ` (i.e. PVC, StorageClass, etc)

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 5]
```diff
@@ -151,6 +163,14 @@ How to use `VolumeSnapshotClass` and `VolumeSnapshot` objects is elaborated on i
 | ----------- | ------  | ----------- |
 | read_only   | Boolean | Indicates if the snapshot is writable on the 3PAR or Primera array. |
 
+### Import Snapshot
+During the import snapshot process, any legacy (non-container snapshot) or an existing docker/k8s snapshot defined in the **ImportVol** parameter, within a StorageClass, will be renamed with the prefix "snapshot-". The new snapshot will be exposed through the HPE CSI Driver and made available to the Kubernetes cluster. **Note:** All previous Access Control Records and Initiator Groups will be removed from the snapshot when it is imported.
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:167]: small tweak: highlight Kubernetes objects with ticks ` (i.e. StorageClass, etc)

File: docs/container_storage_provider/hpe_3par_primera/index.md (hunk 4 above)
Comment (ansarars) [reply docs/container_storage_provider/hpe_3par_primera/index.md]: Highlighted kubernetes objects

File: docs/container_storage_provider/hpe_3par_primera/index.md (hunk 5 above)
Comment (ansarars) [reply docs/container_storage_provider/hpe_3par_primera/index.md]: Highlighted kubernetes objects

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 6]
```diff
@@ -141,6 +142,17 @@ These parameters are applicable only for Pod inline volumes and to be specified
...
+### Import volume
+During the import volume process, any legacy (non-container volumes) or existing docker/k8s volume defined in the **ImportVol** parameter, within a `StorageClass`, will be renamed to match the `PersistentVolumeClaim` that leverages the `StorageClass`. The new volumes will be exposed through the HPE CSI Driver and made available to the Kubernetes cluster. **Note:** All previous Access Control Records and Initiator Groups will be removed from the volume when it is imported.
+
+| Parameter          | Option  | Description |
+| ------------------ | ------- | ----------- |
+| accessProtocol     | fc or iscsi  | The access protocol to use when accessing the persistent volume. |
+| importVol          | Text    | The name of the 3PAR or Primera volume to import. |
+
+!!! important
+    **No other parameters** are required in `StorageClass` for import volume **except** parameters mentioned in above table.
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:154]: grammar: required in the `StorageClass` when importing a volume outside of those parameters listed in the table above.

Comment (ansarars) [reply docs/container_storage_provider/hpe_3par_primera/index.md]: Addressed review comments

**Pull request merged**